oidc/viewsets.py

```diff
@@ -81,86 +85,103 @@ def logout(self, request: HttpRequest, **kwargs: dict) -> HttpResponse:
             _("Unable to process OpenID connect logout request."),
         )
 
+    def _check_user_exists(self, user_data: dict) -> bool:
+        """
+        Helper function that checks if a user exists
+        """
+        if user_data.get(self.unique_user_filter_field):
+            field_value = user_data.get(self.unique_user_filter_field)
+            field = self.unique_user_filter_field + "__iexact"
+            return self.user_model.objects.filter(**{field: field_value}).count() > 0
+        return True
+
+    def generate_successful_response(self, request, user) -> HttpResponse:
+        """
+        Generates a success response for a successful Open ID Connect
+        Authentication request
+        """
+        response = HttpResponseRedirect(config.get("REDIRECT_AFTER_AUTH"))
+
+        if self.use_auth_backend:
+            login(
+                request, user, backend=self.auth_backend,
+            )
```

Comment:
It is about 60 characters long, I think it will still be very readable in a single line and less wordy.
```python
        if self.use_auth_backend:
            login(request, user, backend=self.auth_backend)
```

Reply:
Updated in latest commits